src/test/java/core/basesyntax/service/impl/ReportGeneratorImplTest.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.dao.FruitStorageDaoImpl;
+import core.basesyntax.service.ReportGenerator;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class ReportGeneratorImplTest {
+    private static ReportGenerator reportGenerator;
+    private static FruitStorageDao fruitStorageDao;
+
+    @BeforeAll
+    static void beforeAll() {
+        fruitStorageDao = new FruitStorageDaoImpl();
+        reportGenerator = new ReportGeneratorImpl(fruitStorageDao);
+    }
+
+    @Test
+    void reportGenerator_checkReport_Ok() {
+        fruitStorageDao.updateFruitQuantity("banana", 50);
+        fruitStorageDao.updateFruitQuantity("orange", 30);
```

Comment:
do not use **fruitStorageDao** - cange storage ballance dierectly. 
Unit tests should check only one module. ReportGeneratorImplTest can fail if FruitStorageDaoImpl contains a mistake in its code - this is not correct